### ISSUE TITLE
problem with _events in the mini js file

### ISSUE THREAD

**Issue by natusdrew** [peers/peerjs#675]
Uncaught TypeError: Cannot set property '_events' of undefined
    at i (peerjs.min.js:sourcemap:46)
    at i (peerjs.min.js:sourcemap:66)
    at (index):15

```
<script src="https://unpkg.com/peerjs@1.2.0/dist/peerjs.min.js"></script>
<script>
 14     //try {
 15       var peer = Peer();
 16       console.log("load peer");
 17       console.log(peer);
 18     //}
 19     //catch(err) {
 20     //  console.log(err.message);
 21     //}
 22   </script>```

**Comment by Florrr** [comment]
you are missing the `new`
```
<script src="https://unpkg.com/peerjs@1.2.0/dist/peerjs.min.js"></script>
<script>
    var peer = new Peer();
    console.log("load peer");
    console.log(peer);
</script>
```

**Comment by natusdrew** [comment]
omg. You are right. Let me try again. Will post response.
